charts/zaakbrug/values.yaml

```diff
@@ -116,24 +124,24 @@ zaakbrug:
     #               example: {datetime:[Y001]} with datetime=14-03-2023 produces '2023'
     zaakIdentificatieTemplate: ZK{datetime:[Y0001]}-{id:D5}
     documentIdentificatieTemplate: DC{datetime:[Y0001]}-{id:D5}
+    # Make sure that all Url's contain two "parts" e.g. `openzaak-nginx.zaakbrug`. Openzaak can't use a single part domain.
     zakenApi:
-      rootUrl: "http://open-zaak/zaken/api/v1/"
+      rootUrl: "http://openzaak-nginx.zaakbrug/zaken/api/v1/"
       authType: jwt # Options: 'jwt', 'basic', 'value'. 'value' uses the password field of the given authAlias as Authorization header
       authAlias: zaken-api.jwt # reference to an auth alias configured in credentials.properties
     catalogiApi:
-      rootUrl: "http://open-zaak/catalogi/api/v1/"
+      rootUrl: "http://openzaak-nginx.zaakbrug/catalogi/api/v1/"
       authType: jwt
       authAlias: zaken-api.jwt
     documentenApi:
-      rootUrl: "http://open-zaak/documenten/api/v1/"
+      rootUrl: "http://openzaak-nginx.zaakbrug/documenten/api/v1/"
```

Comment:
This set of root-url's refer to an external OpenZaak instance. Not the staging one. These properties stay as they were

Reply:
I changed them, thinking the extra dot would be useful as a reminder. But I intentionally didn't use `zaakbrug-openzaak-nginx.zaakbrug`

Reply:
I misread it then! having the dot in there is indeed useful as hint. Just need need to make sure it is not mistaken for the staging instance